web/js/jquery.websocket.js

```diff
@@ -35,7 +35,12 @@
                     .bind('message', function(e) {
                         var m = JSON.parse(e.originalEvent.data);
                         var h = settings.events[m.type];
-                        if (h) h.call(this, m);
```

Comment:
I moved the event call into socket.message, so that I could add the desired error messages. settings.events is no longer populated